schema.json

```diff
@@ -4,24 +4,20 @@
     "additionalProperties": false,
     "definitions": {
         "address": {
-            "description": "An address.",
-            "minLength": 1,
-            "type": "string"
+            "$ref": "#/definitions/simplestring",
+            "description": "An address."
         },
         "alias": {
-            "description": "An alias.",
-            "minLength": 1,
-            "type": "string"
+            "$ref": "#/definitions/simplestring",
+            "description": "An alias."
```

Comment:
When I was going through the schema after some time away from it, I also noticed that many of the descriptions under `definitions` are not that helpful. However, with one of the new PRs I made, the fact that these are not part of the user-facing part of the schema is made explicit in that they are under `$defs` instead of `definitions` so maybe not such a big deal.

Reply:
The oneOf join for `person` or `entity`  is not needed AFAIC, I see the 'or' part as a bit of a red flag -- each definition should ideally do one thing.

Reply:
I'll update the CHANGELOG

Reply:
https://github.com/citation-file-format/citation-file-format/pull/466/commits/b4d7bccd3f3b95dbbca9d96343b5e5974ee2372d updated changelog